tests/unit/Dashboard.spec.ts

```diff
@@ -165,12 +193,64 @@ describe('Dashboard - UI', () => {
     expect(localWrapper.find('#standalone-directors-button').attributes('disabled')).toBe('true')
   })
 
-  it('Alert does not display by default, and does display for business not in good standing', async () => {
-    expect(wrapper.find('#dashboard-alerts-section').exists()).toBe(false)
-    wrapper.setData({ isInGoodStanding: false })
-    await Vue.nextTick()
+  it('displays the Amalgamation alert', async () => {
+    // verify initially hidden
+    expect(wrapper.findComponent(Amalgamation).exists()).toBe(false)
+
+    // enable and verify displayed
+    await wrapper.setData({ isFutureEffectiveAmalgamation: true })
+    expect(wrapper.findComponent(Amalgamation).exists()).toBe(true)
+
+    // cleanup
+    await wrapper.setData({ isFutureEffectiveAmalgamation: false })
+  })
+
+  it('displays the Frozen Information alert', async () => {
+    // verify initially hidden
+    expect(wrapper.findComponent(FrozenInformation).exists()).toBe(false)
+
+    // enable and verify displayed
+    await wrapper.setData({ isAdminFrozen: true })
+    expect(wrapper.findComponent(FrozenInformation).exists()).toBe(true)
+
+    // cleanup
+    await wrapper.setData({ isAdminFrozen: false })
+  })
+
+  it('displays the Missing Information alert', async () => {
+    // verify initially hidden
+    expect(wrapper.findComponent(MissingInformation).exists()).toBe(false)
+
+    // enable and verify displayed
+    await wrapper.setData({ hasMissingInfoWarning: true })
+    expect(wrapper.findComponent(MissingInformation).exists()).toBe(true)
+
+    // cleanup
+    await wrapper.setData({ hasMissingInfoWarning: false })
+  })
+
+  it('displays the Not In Compliance alert', async () => {
+    // verify initially hidden
+    expect(wrapper.findComponent(NotInCompliance).exists()).toBe(false)
+
+    // enable and verify displayed
+    await wrapper.setData({ hasComplianceWarning: true })
+    expect(wrapper.findComponent(NotInCompliance).exists()).toBe(true)
+
+    // cleanup
+    await wrapper.setData({ hasComplianceWarning: false })
+  })
+
+  it('displays the Not In Good Standing alert', async () => {
+    // verify initially hidden
+    expect(wrapper.findComponent(NotInGoodStanding).exists()).toBe(false)
+
+    // enable and verify displayed
+    await wrapper.setData({ isGoodStanding: false })
+    expect(wrapper.findComponent(NotInGoodStanding).exists()).toBe(true)
 
-    expect(wrapper.find('#dashboard-alerts-section').exists()).toBe(true)
+    // cleanup
+    await wrapper.setData({ isGoodStanding: true })
   })
```

Comment:
^^ I tried to do these with an array and a for loop but it got messy because the NIGS test requires the getter to be False (opposite of the others). This seemed cleaner.